tests/bootstrap.php

```diff
@@ -195,7 +195,7 @@ function loadDataset() {
             'password2'     => TU_PASS,
             'entities_id'   => '_test_root_entity',
             'profiles_id'   => 4, // TODO manage test profiles
-            '_entities_id'  => '_test_root_entity',
+            '_entities_id'  => 0,
```

Comment:
`_test_root_entity` is a child of the main root entity with id 0.

Reply:
This `_entities_id` is used to create an entry into `glpi_profiles_users`.

Currently, `'_entities_id'  => '_test_root_entity',` is intentionnaly not computed (due to this check: `if (substr($foreigntypetxt, 0, 1) !== '_') {`). Result is that value is replaced by `0`, which is the default value of `glpi_profiles_users.entities_id`. MySQL warning was `` Incorrect integer value: '_test_root_entity' for column `glpi_profiles_users`.`entities_id` at row 1 ``.

I first tried to fix the computation of `_entities_id` to get the id of `_test_root_entity`, but result was that many tests failed.
I can try to fix this specific point on another commit, to see if impact can be acceptable.

Reply:
Oh OK, I had in mind the value stored will change, but this is the exact inverse. So, this change is ok, np ;)